l10n/de/browser/chrome/browser/aboutPrivateBrowsing.dtd

```diff
@@ -5,11 +5,11 @@
 <!ENTITY % brandDTD SYSTEM "chrome://branding/locale/brand.dtd">
```

Comment:
We don't use strings from this file anymore, since we replaced Mozilla's page in DB-662.
Can also safely leave these changes though.